src/hemi-sepolia.tokenlist.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "Hemi Token List",
+  "timestamp": "2024-05-07T22:03:02.686Z",
+  "version": {
+    "major": 1,
+    "minor": 0,
+    "patch": 0
+  },
+  "tokens": [
+    {
+      "address": "0xec46E0EFB2EA8152da0327a5Eb3FF9a43956F13e",
+      "chainId": 743111,
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/ethereum-optimism/ethereum-optimism.github.io/master/data/DAI/logo.svg",
+      "name": "Hemi Tunneled DAI",
+      "symbol": "DAI",
+      "extensions": {
+        "birthBlock": 71589,
+        "bridgeInfo": {
+          "1": {
+            "tokenAddress": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+            "originBridgeAddress": "0xc94b1BEe63A3e101FE5F71C80F912b4F4b055925",
+            "destBridgeAddress": "0x4200000000000000000000000000000000000010"
+          }
+        }
+      }
+    },
+    {
+      "address": "0x0C8aFD1b58aa2A5bAd2414B861D8A7fF898eDC3A",
+      "chainId": 743111,
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/ethereum-optimism/ethereum-optimism.github.io/master/data/WETH/logo.png",
+      "name": "Wrapped Ether",
+      "symbol": "WETH",
+      "extensions": {
+        "birthBlock": 195484
+      }
+    },
+    {
+      "address": "0x3Adf21A6cbc9ce6D5a3ea401E7Bae9499d391298",
+      "chainId": 743111,
+      "decimals": 6,
+      "extensions": {
+        "birthBlock": 575834
+      },
+      "logoURI":
+      "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xdAC17F958D2ee523a2206206994597C13D831ec7/logo.png",
+      "name": "Tether",
+      "symbol": "USDT.e"
+    },
```

Let's use the URLs from optimism, and adding sepolia token address

```suggestion
      "address": "0x3Adf21A6cbc9ce6D5a3ea401E7Bae9499d391298",
      "chainId": 743111,
      "decimals": 6,
      "extensions": {
        "birthBlock": 575834,
        bridgeInfo: {
          "11_155_111": {
            tokenAddress: '0xaa8e23fb1079ea71e0a56f48a2aa51851d8433d0',
          },
        },
      },
      "logoURI":
      "https://raw.githubusercontent.com/ethereum-optimism/ethereum-optimism.github.io/master/data/USDT/logo.png",
      "name": "Tether",
      "symbol": "USDT.e"
    },
```